### PR TITLE
Wildfly 2.0.1

### PR DESCRIPTION
**Pull request by ehsavoie**
### What does this PR do?:
Upgrade WildFly stack

- [X] Contributing guide
_Have you read the [devfile registry contributing guide](https://github.com/devfile/registry/blob/main/CONTRIBUTING.md) and followed its instructions?_
- [X] Test automation
_Does this repository's tests pass with your changes?_
- [ ] Documentation
_Does any documentation need to be updated with your changes?_
- [X] Check Tools Provider
_Have you tested the changes with existing tools, i.e. Odo, Che, Console? (See [devfile registry contributing guide](https://github.com/devfile/registry/blob/main/CONTRIBUTING.md) on how to test changes)_

### How to test changes / Special notes to the reviewer:

### CODE REVIEW

**Comment by ehsavoie** [comment]
I don't understand the odo failure as i couldn't reproduce locally

File: stacks/java-wildfly/2.0.1/deploy.yaml [hunk 1]
```diff
@@ -0,0 +1,71 @@
...
+  name: '{{applicationName}}-svc'
+spec:
+  ports:
+    - name: http-8080
+      port: 8080
+      protocol: TCP
+      targetPort: 8080
+  selector:
+    app: '{{applicationName}}'
+----
```
Comment (thepetk) [inline stacks/java-wildfly/2.0.1/deploy.yaml:39]:
```suggestion

```

**Review by thepetk — CHANGES_REQUESTED** [review]
@ehsavoie added a suggestion which will fix the error:

```
Failed to delete inner loop resources: failed to gather resources for
deletion: yaml: line 39: could not find expected ':'
```

**Review by thepetk — APPROVED** [review]
/lgtm

**Pull request merged**